python/pyspark/sql/tests.py

```diff
@@ -3147,6 +3150,14 @@ def test_filtered_frame(self):
         self.assertEqual(pdf.columns[0], "i")
         self.assertTrue(pdf.empty)
 
+    def test_createDataFrame_toggle(self):
+        pdf = self.createPandasDataFrameFromeData()
+        self.spark.conf.set("spark.sql.execution.arrow.enable", "false")
+        df_no_arrow = self.spark.createDataFrame(pdf)
+        self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
```

Comment:
I'd set this to `true` in `finally` just in case the test failed in `df_no_arrow = self.spark.createDataFrame(pdf)` and `spark.sql.execution.arrow.enable` reminds `false` affecting other test cases if I didn't miss something.

Reply:
done. I guess this would make the failure easier to see?

Reply:
Hmmm, I thought the `tearDownClass` was there but it's actually in #18664.  Maybe I should put it in here since that needs some more discussion.